lib/authentication-context.js

```diff
@@ -142,6 +142,12 @@ Object.defineProperty(AuthenticationContext.prototype, 'options', {
   }
 });
 
+Object.defineProperty(AuthenticationContext.prototype, 'extraQP', {
```

Comment:
Please remove this.  It's not related to the current feature and shouldn't go in here.  We need to be *really* careful about how we change the public API.  No changes should happen to the Api without team discussion.  We need to ensure that the API's are largely the same across platforms.